python/mdvtools/dbutils/dbservice.py

```
@@ -147,7 +275,165 @@ def update_file_timestamp(file_id):
+                "can_read": user_project.can_read,
+                "can_write": user_project.can_write,
+                "is_owner": user_project.is_owner
+            }
+
+        except Exception as e:
+            logger.exception(f"Error in UserProjectService: Failed to get permissions: {e}")
+            raise
+
+
+    @staticmethod
+    def remove_user_from_project(user_id: int, project_id: int):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)